src/api/users/getUsers.js

```diff
@@ -1,6 +1,14 @@
 /* @flow */
-import type { Auth, User } from '../../types';
+import type { Auth, ApiResponseSuccess, User } from '../apiTypes';
 import { apiGet } from '../apiFetch';
 
-export default (auth: Auth): Promise<User[]> =>
-  apiGet(auth, 'users', res => res.members, { client_gravatar: true });
+// Documentation at https://zulipchat.com/api/get-all-users
```

Comment:
This looks like prime material for jsdoc. How about making it a jsdoc comment on the exported function?

Doesn't need to be any longer than this, or
```
/** See https://zulipchat.com/api/get-all-users */
```